src/powersystem.jl

```diff
@@ -60,15 +62,194 @@ struct PowerNetwork
         Ybus, Yf_br, Yt_br, Yf_tr, Yt_tr = makeYbus(data)
 
         # bus type indexing
-        ref, pv, pq = bustypeindex(bus, gen)
-
+        ref, pv, pq, bustype = bustypeindex(bus, gen)
+    
         Sbus = assembleSbus(gen, load, SBASE, nbus)
+        
+        new(V, Ybus, data, nbus, ngen, nload, bustype, ref, pv, pq,
+            Sbus)
+    end
+
+end
+
+"""
+    view(PowerNetwork)
+
+Prints power network characteristics.
 
-        new(V, Ybus, data, nbus, ngen, nload, ref, pv, pq, Sbus)
+"""
+
+function view(pf::PowerNetwork)
```

Comment:
That's nice to have this function! I would be in favor of overloading `Base.show` so we could just do `println(pf)`. 
```
import Base: show
function Base.show(io::IO, pf::PowerNetwork)
     ... the code you implemented here
end
```

Reply:
@frapac yes, you're right. I'll change that.